packages/gravity-ui-web/src/sass/05-components/03-organisms/03-articles/_00-related-items.scss

```diff
@@ -0,0 +1,24 @@
+.grav-c-related-items {
+  max-width: 540px;
+  margin-right: auto;
+  margin-left: auto;
+  padding: 0;
+  list-style: none;
+  text-align: center;
+
+  > li {
+    span {
+      @include grav-color-grp-b-apply('color', 'neutral-alt');
```

Comment:
`neutral-alt` does not exist amongst the group B colors, what you want here is `neutral-subtle`.